src/web-rtc-client.js

```diff
@@ -557,15 +570,15 @@ export default class WebRTCClient extends Emitter {
       sessionDescriptionHandlerFactoryOptions: {
         constraints: {
           audio: this._getAudioConstraints(),
-          video: this.video ? this.cameraDeviceId || true : false,
+          video:  this._getVideoConstraints()
```

Comment:
I don't know why `eslint` doesn't warn about the 2 spaces here

Reply:
I Just removed it